### PR TITLE
UI: show "unknown ID: ..." instead of "error getting benchmark" (fix #906)

### LINKED ISSUE #906: UI shows "error getting benchmark" when an invalid ID was provided.
Seen via CI screenshotting:

image

The cause here was an invalid ID. Improve that error message.

### PR DESCRIPTION
**Pull request by jgehrcke**
This is for addressing #906.

### CODE REVIEW

File: conbench/templates/app.html [hunk 1]
```diff
@@ -86,7 +86,7 @@ <h4 class="modal-title"><i class="bi bi-trash3-fill"></i></h4>
             <!-- Categories: success (green), info (blue), warning (yellow), danger (red) -->
             {% for category, message in messages %}
             <div class="alert alert-{{ category }}" role="alert">
-                <i class="bi bi-lightning"></i> {{ message }}
+                <p class="fs-5"><i class="bi bi-exclamation-diamond-fill"></i> {{ message }}</p>
```
Comment (jgehrcke) [inline conbench/templates/app.html:89]: two changes:
- this icon: https://icons.getbootstrap.com/icons/exclamation-diamond-fill/
- this font size: https://getbootstrap.com/docs/5.2/utilities/text/#font-size

**Review by austin3dickey — APPROVED** [review]

**Review by jonkeane — APPROVED** [review]
Am I reading the CSS | HTML right that this modal(ish) dialog will now persist?

**Comment by jgehrcke** [comment]
> Am I reading the CSS | HTML right that this modal(ish) dialog will now persist?

I think the answer is "no". If I understood the question :). The structure for this modal is always there, but typically it's empty. For populating it with content we use the Flask "flash message" paradigm which shows a message in response to the next request, but doesn't show it anymore after that.

**Comment by jonkeane** [comment]
> > Am I reading the CSS | HTML right that this modal(ish) dialog will now persist?
> 
> I think the answer is "no". If I understood the question :). The structure for this modal is always there, but typically it's empty. For populating it with content we use the Flask "flash message" paradigm which shows a message in response to the next request, but doesn't show it anymore after that.

_nods_ got it, it's still an improvement IMO, so let's ship it and see how it looks. Tangentially | separately the quickness with which some of these error dialogs go away has been a bit confusing — but we can address that separately.

**Comment by jgehrcke** [comment]
> nods got it, it's still an improvement IMO, so let's ship it and see how it looks. Tangentially | separately the quickness with which some of these error dialogs go away has been a bit confusing — but we can address that separately.

Oh, is that about making the modal slowly fade out, controlled by JavaScript? I removed that before I think in the larger PR for the migration towards Bootstrap 5. That is, that info/danger modal content persists until one loads a new page or hits F5 or so. W/o further user interaction, it now stays forever.

**Comment by jonkeane** [comment]
> Oh, is that about making the modal slowly fade out, controlled by JavaScript? I removed that before I think in the larger PR for the migration towards Bootstrap 5. That is, that info/danger modal content persists until one loads a new page or hits F5 or so. W/o further user interaction, it now stays forever.

AAAAH ok, ok got it! Yeah I haven't bumped into it recently, so I must have missed that. Thank you! 🙏

**Pull request merged**